src/wings_frame.erl

```diff
@@ -109,8 +109,13 @@ export_layout() ->
     AddProps = fun({_, _, _}=Split) -> Split;
 		  ({Win, Pos, Size, []}) -> {Win, Pos, Size, window_prop(Win)}
 	       end,
+    AddFreeProps = fun({Win, Pos, _, []}) ->
+		       WxWin = wings_wm:wxwindow(Win),
+		       Size = wxWindow:getClientSize(WxWin),
+		       {Win, Pos, Size, window_prop(Win)}
+		   end,
     Contained = filter_contained(Contained0, AddProps, []),
```

Comment:
Instead of adding a special case here, I think it's better to fix
export_loose()  and let it return the client size directly.

Move Size = wxWindow:getClientSize(WxWin), to export_loose() function.

Reply:
Thanks for the review. Much better!
Nothing better than know the entire code to point exactly where the change need to be done. :-)